src/cloud-sql-instance.ts

```diff
@@ -13,13 +13,15 @@
 // limitations under the License.
 
 import {IpAddressTypes, selectIpAddress} from './ip-addresses';
+import {GoogleAuth} from 'google-auth-library';
```

Comment:
nit: can we get rid of this import now that we moved auth to the connector level?
```suggestion
```